test/blockchain_helpers.cdc

```diff
@@ -0,0 +1,67 @@
+import Test
+
+/// Returns the Flow token balance for the given account.
```

Comment:
Just an idea, not necessarily much better, feel free to ignore:

Maybe group the functions into a struct (maybe `BlockchainHelpers`?), and make the blockchain a field.

That way the `Blockchain` only needs to be passed once, and call-sites of the helper functions become shorter (e.g. `helpers.getFlowBalance(for: 0x1)`)

Reply:
Great idea :100: I will try it out and see how it turns out :+1:

Reply:
That looks indeed way much better:

```cadence
import Test
import BlockchainHelpers

pub let blockchain = Test.newEmulatorBlockchain()
pub let helpers = BlockchainHelpers(blockchain: blockchain)
pub let account = blockchain.createAccount()

// Retrieve current block height
pub let height = helpers.getCurrentBlockHeight()

// Retrieve the Flow token balance of a test account
pub let balance = helpers.getFlowBalance(for: account)

// Mint a given amount of Flow tokens to a specified test account
helpers.mintFlow(to: account, amount: 1500.0)

// Burn a given amount of Flow tokens from a specified test account
helpers.burnFlow(from: account, amount: 500.0)
```
Updated in https://github.com/onflow/cadence-tools/pull/181/commits/7af2b073620f9bb97083cd84d7ab4a8381ab70fb